src/docs/admin/references/NotificationServiceReference.adoc

```diff
@@ -295,3 +295,204 @@ and if they have a value depending on the event origin.
 | Y
 | Y
 |=====
+
+[[microsoft_teams_configuration]]
+==== Microsoft Teams notification method
+
+ProActive provides a ready to use Third-Party script that allows users to send notifications to multiple Teams channels.
+
+The script interacts with Microsoft Graph API to:
+
+ - Connect as a user
+ - Send a message to multiple teams channels
+
+Some configuration must be done in ProActive and Microsoft beforehand but it consists in a nutshell of:
+
+- Configuring the notification service to execute the correct script with the correct setup
+- Create or retrieve app information from the Microsoft admin center
+- Provide these information to the script
+- Update User Settings in the notification portal
+- Add channel URLs to subscriptions
+
+We will go into details on how to achieve this configuration now.
```

Comment:
Maybe to add the list of steps here like:

1. Configure the notification service to execute the correct script with the correct setup.
2. Create or retrieve app information from the Microsoft admin center.
3. Provide this information to the script.
4. Update User Settings in the notification portal.
5. Add channel URLs to subscriptions.